src/components/Module.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from "@emotion/react";
+import { Theme } from "@mui/material";
+import parse from "html-react-parser";
+
+import Card from "./Card";
+
+export type ModuleProps = {
+  analysis: ReturnType<typeof parse> | null;
+};
+
+const Module: React.VFC<ModuleProps> = (props) => {
+  const { analysis } = props;
+  return (
+    <Card header="Module" css={styles}>
+      <div className="module-content">{analysis}</div>
+    </Card>
+  );
+};
+export default Module;
+
+const styles = (theme: Theme) => css`
+  .module-content {
```

Comment:
@Stinggyray this was the style block I was talking about earlier

Reply:
I think when making this I was like "I might need styles"
And then I just didn't need styles